apps/build.groovy

```diff
@@ -50,9 +27,8 @@ class AppBuildResults implements Serializable {
 def build(String build_env) {
 	dir ('apps') {
 
-		mvn "--update-snapshots -Dmaven.test.failure.ignore clean verify"
+		sh "mvn --update-snapshots -Dmaven.test.failure.ignore clean verify"
```

Comment:
Since all maven settings and toolchain configs are embedded into our build environment container, we no longer need this and can run maven directly.  I think this is acceptable because the only time we should ever have to update maven settings or toolchains is if we bump a major version of java or maven, which would require a container rebuild anyway.